downstream/modules/platform/proc-install-aap-operator.adoc

```diff
@@ -4,7 +4,11 @@
 . Log in to {OCP}.
 . Navigate to *Operators* -> *OperatorHub*.
 . Search for the {PlatformName} operator and click *Install*.
-. Select an *Installation Mode*, *Installed Namespace*, and *Approval Strategy*.
+. Select an *Update Channel*:
++
+* *stable-2.x*: deploys only one {HubName} or {ControllerName} instance and is suitable for most cases. The stable-2.x channel does not require administrator privileges and utilizes fewer resources because it only monitors a single namespace.
```

Comment:
@rooftopcellist 
Can we say:
* *stable-2.x*: limits deployments of {HubName} and {ControllerName} instances to a single namespace and is suitable for most cases. The stable-2.x channel does not require administrator privileges and utilizes fewer resources because it only monitors a single namespace.

Reply:
@rooftopcellist @lmalivert @ariordan-redhat , since this PR is already merged, I will make this change in a separate PR.

Reply:
@ariordan-redhat,  @dcdacosta and I modified it a bit more to make it clearer.  We ended up with:

* _stable-2.x:_ installs a namespace-scoped operator, which limits deployments of {HubName} and {ControllerName} instances to the namespace the operator is installed in. This is suitable for most cases. The stable-2.x channel does not require administrator privileges and utilizes fewer resources because it only monitors a single namespace.